### ISSUE TITLE
feeds should work on tag pages as well

### ISSUE THREAD

**Issue by daonb** [hasadna/open-shot#36]

**Comment by pwalsh** [comment]
meaning, all questions with this tag?

**Comment by daonb** [comment]
yep, but it can also be a list of comma seperated tags. I suggest you add a get a get parameter `tags` with a comma seperated list of tags and filter by it. check `okqa.qa.views.tagged_questions` for code to copy.

**Comment by pwalsh** [comment]
Ok, so a tag view can be for one or many tags. If many, then the keyword
argument is a comma separated list of the tags. Correct?

**Comment by daonb** [comment]
Correct

**Comment by daonb** [comment]
I suggest we use the chance to refactor the feed view functions to feeds.py
